tox.ini

```diff
@@ -84,7 +84,7 @@ envlist =
     elasticsearch_contrib-{py27,py35,py36}-elasticsearch2{50}
     elasticsearch_contrib-{py27,py35,py36}-elasticsearch5{50}
     elasticsearch_contrib-{py27,py35,py36}-elasticsearch6{40}
-    falcon_contrib{,_autopatch}-{py27,py35,py36}-falcon{10,11,12,13,14}
+    falcon_contrib{,_autopatch}-{py27,py35,py36,py37}-falcon{10,11,12,13,14,20}
```

Comment:
Is Python 3.8 also supported by falcon?

Reply:
I don't believe so since the [2.0 change logs](https://falcon.readthedocs.io/en/stable/changes/2.0.0.html#summary) only mention Python 3.7. It seems they support Python 3.8 in [Falcon 3.0](https://falcon.readthedocs.io/en/latest/changes/3.0.0.html) but that hasn't been released yet